common/src/com/intellij/plugins/haxe/compilation/HaxeCompilerError.java

```diff
@@ -79,11 +79,11 @@ public static HaxeCompilerError create(@NotNull String rootPath,
     {
         Matcher m;
 
-        // Error: Library ([^\w]+) is not installed(.*)
```

Comment:
The comment did not match the regular expression.  I also eliminated the "Error:" portion of the expression so that it would match the hxcpp error as well as the haxe one.